modules/retail/pages/retail-install-unified.adoc

```diff
@@ -84,22 +84,20 @@ For more information about activation keys, see xref:client-configuration:client
 * SLE-Module-Server-Applications15-SP2-Updates for x86_64 SMRBS 4.1
 * SLE-Product-SUSE-Manager-Retail-Branch-Server-4.1-Updates for x86_64
 . Use this activation key in {susemgrproxy} registration at xref:installation:proxy-registration.adoc[].
-
-
++
 [WARNING]
 ====
-Do not configure the branch server as a traditionally managed proxy.
-The branch server must be configured as a Salt client.
+The branch server must be configured as a Salt managed proxy.
 ====
-
+. Configure {susemgrproxy} as instructed at xref:installation:proxy-setup.adoc[].
```

Comment:
Can we move this step up into the procedure proper, and then take the two admonitions out of the procedure and just have them underneath?

Reply:
I had that in early version, but I thought this would be better to indicate what is the warning related to. I'll update it.

Reply:
Yeah, having the admons in the procedure kinda breaks the flow pretty badly. It remains to be seen if the new procedure/admon styling will make it suck less.